teams/lang-ops.toml

```
@@ -0,0 +1,18 @@
+name = "lang-ops"
+subteam-of = "lang"
+
+[people]
+leads = ["TC"]
+members = [
+    "TC",
+]
```

```suggestion
leads = ["traviscross"]
members = [
    "traviscross",
]
```

like wg-async:
https://github.com/rust-lang/team/blob/765876e23d928b3934a6b07855ed741c4dd0b837/teams/wg-async.toml#L17

---

Huh, OK, we use github names I guess, and not the name of the "people" file?